### PR TITLE
Marcos.reginato/visor design

### PR DESCRIPTION
**Pull request by regmarcos**
Hi, i made some little changes on the presenter and the main_activity layout.
Now you can use the app only as portrait screen.

Anotación PR2

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -2,32 +2,39 @@
...
-        android:layout_height="@dimen/visor_operation"
-        android:textAlignment="gravity"
+        android:layout_height="@dimen/visor_operation_dim"
+        android:layout_marginTop="@dimen/marginOperation"
+        android:layout_marginLeft="@dimen/marginOperation"
+        android:textSize="@dimen/textOperationSize"
         app:layout_constraintBottom_toTopOf="@id/visor_result"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="SpUsage"
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: Please, modify this why are you being recommended to use SP style to format a textSize

**Review by SartiMau — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (regmarcos) [reply app/src/main/res/layout/activity_main.xml]: done

**Review by mauroLonghi — APPROVED** [review]

**Comment by regmarcos** [comment]
The layout with new colors

Anotación 2019-11-07 130315

File: app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java [hunk 2]
```diff
@@ -3,12 +3,8 @@
 import com.example.calculadoramvp.mvp.model.CalculatorModel;
 import com.example.calculadoramvp.mvp.view.CalculatorView;
 import android.widget.Toast;
-import static com.example.calculadoramvp.utils.Utils.EMPTY_STRING;
-import static com.example.calculadoramvp.utils.Utils.MINUS;
-import static com.example.calculadoramvp.utils.Utils.MULTIPLICATION;
-import static com.example.calculadoramvp.utils.Utils.PLUS;
-import static com.example.calculadoramvp.utils.Utils.TOAST_ERROR_MSG;
-import static com.example.calculadoramvp.utils.Utils.TOAST_MSG_FIRST_OPERAND;
+import static com.example.calculadoramvp.utils.Utils.*;
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java]: don't use wildcard imports pls

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -37,6 +39,7 @@
         android:layout_width="@dimen/button_layout_width"
         android:layout_height="@dimen/button_layout_height"
         android:text="@string/zero"
+        android:background="#81c8c0"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: user colors resources pls

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java (hunk 2 above)
Comment (regmarcos) [reply app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java]: done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (regmarcos) [reply app/src/main/res/layout/activity_main.xml]: done

**Review by SartiMau — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Pull request merged**